src/callDataService.ts

```
@@ -0,0 +1,33 @@
+
+export async function appendTranscription(botId: string, newText: string) {
+  try {
+    const [rows]: any = await db.execute(
+      "SELECT transcription FROM call_data WHERE Bot_Id = ?",
+      [botId]
+    );
+
+    const currentText = rows.length > 0 ? rows[0].transcription || "" : "";
+    const updatedText = currentText + " " + newText;
+
+    await db.execute(
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)